cassandra-migration-spring-boot-starter/src/main/java/org/cognitor/cassandra/migration/spring/CassandraMigrationConfigurationProperties.java

```diff
@@ -14,33 +18,50 @@
 @ConfigurationProperties(prefix = "cassandra.migration")
 public class CassandraMigrationConfigurationProperties {
     private ScriptCollectorStrategy strategy = ScriptCollectorStrategy.FAIL_ON_DUPLICATES;
-    private String scriptLocation = MigrationRepository.DEFAULT_SCRIPT_PATH;
+    private List<String> scriptLocations = Collections.singletonList(MigrationRepository.DEFAULT_SCRIPT_PATH);
     private String keyspaceName;
     private String tablePrefix = "";
     private String executionProfileName = null;
     private DefaultConsistencyLevel consistencyLevel = DefaultConsistencyLevel.QUORUM;
     private Boolean withConsensus = false;
 
-    /**
-     * @return The location of the migration scripts. Never null.
-     */
-    public String getScriptLocation() {
-        return scriptLocation;
-    }
-
     /**
      * The location where the scripts reside on the classpath.
      * The default is <code>MigrationRepository.DEFAULT_SCRIPT_PATH</code> which
      * points to <code>cassandra/migration</code> on the classpath.
+     * @deprecated Use {@link #setScriptLocations(List)} instead.
      *
      * @param scriptLocation the location of the migration scripts. Must not be null.
      * @throws IllegalArgumentException when scriptLocation is null or empty
      */
+    @Deprecated
     public void setScriptLocation(String scriptLocation) {
         if (scriptLocation == null || scriptLocation.isEmpty()) {
             throw new IllegalArgumentException("Script location cannot be unset.");
         }
-        this.scriptLocation = scriptLocation;
+        this.scriptLocations = Collections.singletonList(scriptLocation);
+    }
+
+    /**
+     * @return The locations of the migration scripts. Never null.
+     */
+    public List<String> getScriptLocations() {
+        return scriptLocations;
```

Comment:
I would suggest to return `Collections.unmodifiableList(scriptLocations)` to prevent any tempering with the list.

Reply:
Sure, thanks for the suggestion